gpustack/worker/tools_manager.py

```diff
@@ -445,16 +456,20 @@ def _download_llama_box(
 
         platform_name = self._get_llama_box_platform_name(version)
         tmp_file = llama_box_tmp_dir / f"llama-box-{version}-{platform_name}.zip"
-        url_path = f"gpustack/llama-box/releases/download/{version}/llama-box-{platform_name}.zip"
-
+        # Since v0.0.157: Use dynamic libraries instead of static linking
+        if version >= "v0.0.157":
+            url_path = f"gpustack/llama-box/releases/download/{version}/dl-llama-box-{platform_name}.zip"
+        else:
+            url_path = f"gpustack/llama-box/releases/download/{version}/llama-box-{platform_name}.zip"
```

Comment:
Let's nest the `dl-llama-box-` or `llama-box-` in where we get the `platform_name`.